controller-server/src/test/java/com/yahoo/vespa/hosted/controller/ControllerTest.java

```diff
@@ -633,6 +637,14 @@ public void testDevDeployment() {
         assertTrue("No job status added",
                    context.instanceJobs().isEmpty());
         assertEquals("DeploymentSpec is not persisted", DeploymentSpec.empty, context.application().deploymentSpec());
+
+        // Verify routingMethod supports shared layer 4 and shared
+        Set<RoutingMethod> routingMethods = tester.controller().routing().endpointsOf(context.deploymentIdIn(zone))
+                .asList()
+                .stream()
+                .map(Endpoint::routingMethod)
+                .collect(Collectors.toSet());
+        assertThat(routingMethods, IsIterableContainingInAnyOrder.containsInAnyOrder(RoutingMethod.shared, RoutingMethod.sharedLayer4));
```

Comment:
Isn't this equivalent to `assertEquals(Set.of(RoutingMethod.shared, RoutingMethod.sharedLayer4), routingMethods)`?